apps/mdreader/app/routes/knowledge-groups.$slug._index.tsx

```diff
@@ -82,28 +92,31 @@ const KnowledgeCard: React.FC<KnowledgeCardProps> = ({
 
 type OutletContext = {
   knowledgeGroup: KnowledgeGroup;
+  session: Session | null;
 };
 
 const KnowledgeGroup = () => {
-  const { knowledgeGroup } = useOutletContext<OutletContext>();
+  const { knowledgeGroup, session } = useOutletContext<OutletContext>();
   const { data, isLoading, mutate } = useKnowledgeBases(knowledgeGroup.id);
```

Comment:
**Suggestion:** Consider using a more descriptive variable name than 'data'. It could be 'knowledgeBasesData' to make it clear what data it holds.
```suggestion
  const { data: knowledgeBasesData, isLoading, mutate } = useKnowledgeBases(knowledgeGroup.id);
```